README.md

```diff
@@ -1,40 +1,58 @@
 # OSS-Fuzz: Continuous Fuzzing for Open Source Software
```

Comment:
Please also modify https://github.com/google/oss-fuzz/blob/master/docs/index.md in a similar way. it is the same content.